forge-game/src/main/java/forge/game/spellability/AbilityStatic.java

```diff
@@ -52,6 +59,15 @@ public AbilityStatic(final Card sourceCard, final Cost abCost, final TargetRestr
     public boolean canPlay() {
         final Card c = this.getHostCard();
 
+        // Check if ability can't be attempted because of replacement effect
+        // Initial usage is Karlov Watchdog preventing disguise/morph/cloak/manifest turning face up
+        if (this.isTurnFaceUp() && c.isFaceDown()) {
```

Comment:
Is `&& c.isFaceDown()` even necessary?

Reply:
should not be needed i think

the Abilities are only added when the Card is face down, or have a Condition for Facedown